lib/widgets/row_genres.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+import '../utils/constants.dart';
+import 'container_genres.dart';
+
+class GenresRow extends StatelessWidget {
+  const GenresRow({
+    super.key,
+  });
+
```

Comment:
Thinking a bit ahead of time, I'd make this widget receive the genres list!

Reply:
Perfect, i will implement this for the next pr as we discussed.